src/en/news/blog/2023/landing-page/index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Dashboard Landing Page
+date: 2023-02-14
+author: Pedro Gonzalez Gomez
+image: "image.jpg"
+tags:
+  - "Ceph"
+  - "Dashboard"
+---
+
+## Ceph Dashboard and The Landing page
+
+Ceph Dashboard is a built-in web-based management and monitoring application through which you can inspect and administer various aspects and resources within a Ceph cluster. It is implemented as a Ceph Manager Daemon module. Learn more about it [here](https://docs.ceph.com/en/quincy/mgr/dashboard/#overview).
+
+The landing page is the home page of the dashboard, this one displays important metrics such as the overall cluster status, performance, and capacity. It provides real-time updates on changes in the cluster and allows quick access to other sections of the dashboard.
+
+As the Ceph Dashboard continues to add new features and capabilities, the landing page has been updated to keep pace. We are excited to present the updated landing page:
+
+![Landing page](assets/landing-page.png)
+
+## Goals
+
+The goal of this project is to optimize the user experience of the landing page. This will be achieved through a combination of preserving existing features, introducing new ones, presenting information in a clear and organized manner, and crafting a user-friendly interface. By following these guidelines, we aim to make the landing page a powerful tool for monitoring the cluster with ease.
```

Comment:
```suggestion
The goal of this project is to enhance the user experience of the landing page. This will be achieved through a combination of preserving existing features, introducing new ones, presenting information in a clear and organized manner, and crafting a user-friendly interface. By following these guidelines, we aim to make the landing page a powerful tool for monitoring the cluster with ease.
```